config/registry_image_pruner/kustomization.yaml

```diff
@@ -0,0 +1,17 @@
+namespace: image-controller-system
+
```

Comment:
include this file in config/base

Reply:
What do you mean? I don't see config/base

Reply:
I think that `image-controller-system` should be "kustomizable".
In real clusters, usually, the namespace is `image-controller`.

Reply:
I am still lost :D

Reply:
Ok I meant https://github.com/redhat-appstudio/image-controller/blob/main/config/default/kustomization.yaml

Reply:
add `../registry_image_pruner` into bases and test by running `kustomize build config/default` that should generate the CRs also with the cronjob.

Reply:
I did that and also removed namespace and nameprefix from config/registry_image_pruner/kustomization.yaml as they are mentioned in the default and other kustomization files don't have them.